dev-docs/modules/gdprEnforcement.md

```diff
@@ -49,115 +50,101 @@ A page needs to define configuration rules about how Prebid.js should enforce ea
+| gdpr.rules[].purpose | `String` | Supported values: "storage" (Purpose 1), "basicAds" (Purpose 2) | "storage" |
+| gdpr.rules[].enforcePurpose | `Boolean` | Determines whether to enforce the purpose consent. The default in Prebid.js 3.x is not to enforce purposes. Prebid.js 4.0 enforces legal basis for Purposes 1 and 2 by default. | true |
+| gdpr.rules[].enforceVendor | `Boolean` | Determines whether to enforce vendor signals for this purpose. The default in Prebid.js 3.x is not to enforce vendor signals. Prebid.js 4.0 enforces legal basis for Purposes 1 and 2 by default. | true |
 | gdpr.rules[].vendorExceptions | `Array of Strings` | Defines a list of biddercodes or module names that are exempt from the enforcement of this Purpose. | ["bidderA", "userID-module-B"] |
 
 Note:
 
-- The vendorExceptions list is based on Prebid.js biddercodes instead of Global Vendor List (GVL) IDs, i.e. "rubicon" instead of "52". This was done to accomodate Prebid.js modules and adapters that don't have GVL IDs.  
+- To accomodate Prebid.js modules and adapters that don't have GVL IDs, the vendorExceptions list is based on Prebid.js biddercodes instead of Global Vendor List (GVL) IDs (i.e. "rubicon" instead of "52").
 
 ### Examples
 
-The following examples cover a range of use cases and how Prebid.js supports
+The following examples cover a range of use cases and show how Prebid.js supports
 configuration of different business rules.
 
-1) Enforce that the user consents to DeviceAccess as an activity and consider their per-vendor selection.
+1) Enforce device access activity and basic ads. These are the default values (in Prebid.js 4.0) if the module is included in the build.
 
 ```
 pbjs.setConfig({
   consentManagement: {
     gdpr: {
-      ...
-      rules: [{
+      cmpApi: 'iab',   // activates the enforcement module
+      defaultGdprScope: true,
+      rules: [{        // these are the default values
         purpose: "storage",
         enforcePurpose: true,
         enforceVendor: true
+      },{
+        purpose: "basicAds",
+        enforcePurpose: true,
+        enforceVendor: true
       }]
     }
   }
 });
 ```
 
-2) Enforce that the user consents to DeviceAccess as an activity and consider their per-vendor selection. However, BidderA is a special case - the publisher has entrusted BidderA for this activity.
+2) Enforce that the user consents to DeviceAccess as an activity and consider their per-vendor selection. However, idSystemA is a special case - the publisher has confirmed that this system obtains the ID every auction and does not write to the local device.
 
       ...
       rules: [{
         purpose: "storage",
         enforcePurpose: true,
         enforceVendor: true,
-        vendorExceptions: ["bidderA"]
+        vendorExceptions: ["idSystemA"]
```

Comment:
The note above says we don't use IDs. So I find making this an "id" kind of confusing.

Reply:
The note says we don't use GVL IDs. Yeah, too many uses of the phrase "ID", but I don't see a way around this. I tweaked the example text to note "User ID"

Reply:
better, thanks